### ISSUE TITLE
[FVT-Framework] Need more than one minute for xCAT works in sles11.4 ppc64 and x86_64

### ISSUE THREAD

**Issue by caomengmeng** [xcat2/xcat-core#1976]
ENV.: sles11.4 ppc64 and x86_64
Description: after install xCAT with go-xcat, it needs more than one minute for xCAT work.
Steps:
1. install diskful installation in CN.
2. install xCAT with go-xcat tool.
3. run `source "/etc/profile.d/xcat.sh"`
4. run lsxcatd -v, below are the output messages:

```
Connection failure: SSL connect attempt failed with unknown errorerror:00000000:lib(0):func(0):reason(0) at /opt/xcat/lib/perl/xCAT/Client.pm line 278.
Unable to open socket connection to xcatd daemon on localhost:3001.
Verify that the xcatd daemon is running and that your SSL setup is correct.
CHECK:rc == 0   [Failed]
```

**Comment by whowutwut** [comment]
@cxhong  any ideas?

**Comment by tingtli** [comment]
I found on sles11.4 platform, xCAT installation will consume very long time. I use both zypper and go-xcat to install xCAT on sles11.4. I found, if we use zypper, we need to wait about several minutes to get xcat commands up, and for go-xcat on sles11.4, go-xcat hangs there!

**Comment by immarvin** [comment]
cannot recreate, the go-xcat takes about 4 min

**Comment by tingtli** [comment]
@immarvin It seems that it could be created ONLY in the automation environment. Would you please help debug in the automation environment.

**Comment by hu-weihua** [comment]
@caomengmeng , @tingtli , after debug, I found it isn't a bug of xCAT product, it a problem of test case ``install_xCAT_on_rhels_sles``.

Due to command ``source "/etc/profile.d/xcat.sh"`` just write environment attribute to current process space, so if process space is changed, these  environment attributes will gone.

Our test case use xcattest framework, xcattest trigger each command by `` ` ` `` , that means each command will run in independent process space. So below statements won't work as we expected.

     cmd:source "/etc/profile.d/xcat.sh"
     check:rc==0
     cmd:lsxcatd -v
     check:rc==0

This problem will always failed in redhat, sles and ubuntu. Why we didn't found problem in redhat and ubuntu, even in sles 12, that because in ``xcatjktest``we have set up  environment attribute by below code:

     &runcmd("xdsh $mn \"touch /etc/profile.d/xcatjk.sh; echo -e \\\"XCATROOT=/opt/xcat\nPATH=\$XCATROOT/bin:\$XCATROOT/sbin:\$XCATROOT/share/xcat/tools:\$PATH\nMANPATH=\$XCATROOT/share/man:\$MANPATH\nexport XCATROOT PATH MANPATH\nexport PERL_BADLANG=0\\\" >> /etc/profile.d/xcatjk.sh;source /etc/profile.d/xcatjk.sh\" >/dev/null 2>&1");

So even ``install_xCAT_on_rhels_sles`` doesn't include  ``source "/etc/profile.d/xcat.sh"``. All environment attributes needed by redhat, ubuntu and sles12 have taken effect. 

Unfortunately, in sles 11.4, there is specific environment attributes ``XCATSSLVER=TLSv1 ``which used to build ssl connection,   ``xcatjktest`` didn't cover this one, so the above problem was exposed.

So recommend using below command to set up environment attributes

    cmd: source "/etc/profile.d/xcat.sh";lsxcatd -v
